### PR TITLE
add more monitoring tools to `apps_groups.conf`

### PR DESCRIPTION
**Pull request by andrewm4894**
##### Summary

add `datadog`, `edgedelta`, `newrelic`, `google-agent`, `nvidia-smi`, `htop`

tested locally and working as expected.

image

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Review by ilyam8 — APPROVED** [review]

File: collectors/apps.plugin/apps_groups.conf [hunk 1]
```diff
@@ -159,6 +159,12 @@ logs: ulogd* syslog* rsyslog* logrotate systemd-journald rotatelogs
 nms: snmpd vnstatd smokeping zabbix* monit munin* mon openhpid watchdog tailon nrpe
 splunk: splunkd
 azure: mdsd *waagent* *omiserver* *omiagent* hv_kvp_daemon hv_vss_daemon *auoms* *omsagent*
+datadog: *datadog*
+edgedelta: edgedelta
+newrelic: newrelic*
+google-agent: *google_guest_agent* *google_osconfig_agent*
+nvidia-smi: nvidia-smi
+htop: htop
```
Comment (ilyam8) [inline collectors/apps.plugin/apps_groups.conf:167]: Why do you think we need `htop`? It is not a daemon and I think it consumes very few CPU/mem.

Comment (thiagoftsm) [reply collectors/apps.plugin/apps_groups.conf:167]: I had the same doubt, I am waiting for the answer to approve the PR.

Comment (andrewm4894) [reply collectors/apps.plugin/apps_groups.conf:167]: its useful for signaling when you or someone else was working on the system and so using htop locally.

**Review by thiagoftsm — APPROVED** [review]

File: collectors/apps.plugin/apps_groups.conf (hunk 1 above)
Comment (ilyam8) [reply collectors/apps.plugin/apps_groups.conf:167]: This doesn't really answer my question, my doubts:

- not a daemon, but just a tool (I think people use it for several seconds to check something quickly).
- uses little CPU/mem => and doesn't contribute a lot to overall CPU usage => it is ok to have it in "other".

So I don't see how having `htop` as a dimension is any help. My opinion is that the majority of users don't care if somebody using "using htop locally".

**Pull request merged**